assets/js/googlesitekit-idea-hub-post-list.js

```diff
@@ -46,9 +47,31 @@ domReady( () => {
 	const expiresInSeconds =
 		type === 'idea-hub_new-ideas' ? WEEK_IN_SECONDS : 0;
 
-	notice.addEventListener( 'click', ( event ) => {
-		if ( event.target.classList.contains( 'notice-dismiss' ) ) {
+	trackEvent(
+		'idea_hub_posts',
+		type === 'idea-hub_saved-ideas'
+			? 'savedposts_notification_view'
+			: 'newposts_notification_view'
+	);
+
+	notice.addEventListener( 'click', ( { target } ) => {
+		if ( target.classList.contains( 'notice-dismiss' ) ) {
 			dispatch( CORE_USER ).dismissItem( type, { expiresInSeconds } );
+
+			trackEvent(
+				'idea_hub_posts',
+				type === 'idea-hub_saved-ideas'
+					? 'savedposts_notification_dismiss'
+					: 'newposts_notification_dismiss'
+			);
+		} else if (
+			target.classList.contains( 'googlesitekit-idea-hub__saved-ideas' )
+		) {
+			trackEvent( 'idea_hub_posts', 'savedposts_notification_complete' );
+		} else if (
+			target.classList.contains( 'googlesitekit-idea-hub__new-ideas' )
+		) {
+			trackEvent( 'idea_hub_posts', 'newposts_notification_complete' );
```

Comment:
Should we await these since the click here will trigger a navigation?

Reply:
I think that even if we add `await` here, the event listener will not wait till the promise is resolved. Anyway, when I tested it on my local machine, it worked correctly.